lang/bg.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Play": "Възпроизвеждане",
+  "Pause": "Пауза",
+  "Current Time": "Текущо време",
+  "Duration Time": "Продължителност",
+  "Remaining Time": "Оставащо време",
+  "Stream Type": "Тип на излъчването",
+  "LIVE": "НА ЖИВО",
+  "Loaded": "Заредено",
+  "Progress": "Прогрес",
+  "Fullscreen": "Голям екран",
+  "Non-Fullscreen": "Спиране на голям екран",
+  "Mute": "Без звук",
+  "Unmuted": "Със звук",
+  "Playback Rate": "Скорост на възпроизвеждане",
+  "Subtitles": "Субтитри",
+  "subtitles off": "Спри субтитрите",
+  "Captions": "Надписи за глухи",
+  "captions off": "Спри надписите за глухи",
+  "Chapters": "Глави",
+  "You aborted the video playback": "Спряхте възпроизвеждането на видеото",
+  "A network error caused the video download to fail part-way.": "Грешка във мрежата провали свалянето на клипа.",
```

Comment:
'Грешка **в** мрежата' because 'мрежата' does not start with a 'в' or a 'ф'.
'Свалянето' feels too informal. 'Изтеглянето' or 'прехвърлянето' sound more appropriate.